discussion/views.py

```diff
@@ -185,6 +186,19 @@ def create_attachments(request):
     # The user is uploading one or more files.
     ret = { }
     for fn in request.FILES:
+        # Validate before attachment object creation
+        uploaded_file = request.FILES[fn]
+
+        # 2.5MB
+        if sys.getsizeof(uploaded_file) >= 2621440:
```

Comment:
This could be stored as a constant where ever all of the constants are kept for the discussion view